src/client/cl_console.c

```diff
@@ -202,8 +202,6 @@ void Cl_DrawChat(void) {
 			g_free(lines[i]);
 			y += ch;
 		}
-
-		Mem_Free(lines);
```

Comment:
This one in particular (Mem_Malloc vs alloca) is concerning. Mem_Malloc is slow.